Modules/modExtendSaveAsText.ACM

```diff
@@ -1210,6 +1230,80 @@ Private Function IgnoreSection(bytsFile() As Byte, bytsLine() As Byte, lngStartP
   
 End Function
 
+Public Function ReplaceCodeSection(strFilename As String, strNewCodeFilename As String) As Boolean

```

Comment:
`strFilename` should perhaps be `strOrigFilename`, this would also match with the `bytsFileOrig` naming.

Reply:
Agreed, done